sdk/storage/storage-file-share/test/fileclient.spec.ts

```diff
@@ -57,8 +57,10 @@ describe("FileClient", () => {
   });
 
   afterEach(async function() {
-    await shareClient.delete();
-    recorder.stop();
+    if (!this.currentTest?.isPending()) {
```

Comment:
Why this?
`afterEach` would anyway be executed only after the current test ends, right?

Reply:
Well issuing a delete breaks the recording, since the test was skipped and we didn't record that operation. We could also blanketly re-record everything, but it feels weird to do cleanup on skipped tests

Reply:
Ah, got the problem...
Previously, afterEach was skipped if the test is skipped, but with mocha@7.0.0, afterEach gets executed even if the test is skipped.
This is fine here, we can fix these in a followup issue..  https://github.com/Azure/azure-sdk-for-js/issues/8110